internal/util/pkgutil/pkgutil.go

```diff
@@ -15,25 +15,67 @@
 package pkgutil
```

Comment:
This should be folded into Package type. Following comments about the need for better abstractions for a Package and avoiding passing raw strings around everywhere.

Also see:

https://github.com/GoogleContainerTools/kpt/blob/f652b15b033a763967152e397d05320fe53166f4/internal/pkg/pkg.go#L162

Reply:
One suggestion to reduce the size of this PR: Have the changes to the Package type as as separate PR. The remaining code can use the high level Package abstraction.

Reply:
yeah, the `internal/util/pkgutil` and `kpt/internal/pkg` should probably be combined.

I want to get the behavior of get and update correct before trying to use the Package type. Otherwise we will end up spending time refactoring code that we plan to rewrite anyway.